### PR TITLE
Remove XML link in mapinfo command, and refactor command classes

### PR DESCRIPTION
**Pull request by PotatoStealer**
XML links now point to some stratus.nyc3.digitaloceanspaces URL, which is inaccessible. This commit temporarily removes the link displayed in /map until the URL is fixed. This commit also starts the cleanup of the various commands classes in the PGM submodule.

### CODE REVIEW

**Comment by PotatoStealer** [comment]
The localized messages aren't working at the moment(found after testing), I'll fix it in later commits.

File: PGM/src/main/java/tc/oc/pgm/commands/CommandManifest.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+import tc.oc.commons.core.commands.CommandBinder;
+import tc.oc.commons.core.inject.HybridManifest;
+
+public class CommandManifest extends HybridManifest {
+    @Override
+    protected void configure() {
+        new CommandBinder(binder()).register(AdminCommands.class);
+        new CommandBinder(binder()).register(MapCommands.class);
+        new CommandBinder(binder()).register(MatchCommands.class);
```
Comment (RainoBoy97) [inline PGM/src/main/java/tc/oc/pgm/commands/CommandManifest.java]: Only need one `CommandBinder` here?

Comment (PotatoStealer) [reply PGM/src/main/java/tc/oc/pgm/commands/CommandManifest.java]: ✔️

**Comment by Pugzy** [comment]
The map XML links now function correctly.

**Comment by AustinLMayes** [comment]
Too many changes for one PR. Separate them, and recreate multiple requests with each difference change category.

**Pull request closed without merging**